src/environments/environment.prod.ts

```diff
@@ -20,3 +20,24 @@ export const environment: Environment = {
 
   itemPlatformId: 'algorea_backend',
 };
+
+export const presets: Record<string, Partial<Environment>> = {
+  example: {
+    defaultActivityId: '1625159049301502151', // Motif Art
+    defaultTitle: 'Example app',
+    authType: 'tokens',
+    itemPlatformId: 'Example',
+  },
+  demo: {
+    defaultActivityId: '1352246428241737349', // SNT
+    defaultTitle: 'Demo app'
+  },
+};
+
+export function getPresetNameByDomain(domain: string): string | null {
+  switch (domain) {
```

Comment:
I would have expected the record key to be the pattern matcher (possibly a regexp or a list if we want to match several ones). But it can be "fixed" later.